internal/services/formatters/php/phpcs/formatter.go

```diff
@@ -94,20 +96,27 @@ func (f *Formatter) parseResults(results map[string]interface{}) {
 
 func (f *Formatter) parseMessages(filepath string, result interface{}) {
 	for _, message := range f.parseToResult(result).Messages {
-		if message.IsValidMessage() {
-			f.AddNewVulnerabilityIntoAnalysis(f.setVulnerabilityData(filepath, message))
+		if message.IsAllowedMessage() &&
+			!strings.Contains(message.Message, "This implies that some PHP code is not scanned by PHPCS") {
+			vuln := f.setVulnerabilityData(filepath, message)
+			f.AddNewVulnerabilityIntoAnalysis(vuln)
```

Comment:
Turning this contains validation into a method can make it easy to read.

```suggestion
		if message.IsAllowedMessage() && m.IsNotFailedToScan() {
		     ...
		}
		
		func (m *Message)IsNotFailedToScan() {
		      return !strings.Contains(message.Message, "This implies that some PHP code is not scanned by PHPCS")
		}
```

Reply:
done :)